modules/knative-serving-concurrent-autoscaling-requests.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * serverless/configuring-knative-serving-autoscaling.adoc
+
+[id="knative-serving-concurrent-autoscaling-requests_{context}"]
+= Configuring concurrent requests for Knative Serving autoscaling
+
+The default limit for the number of concurrent requests is `100`, which is defined in the Knative Serving ConfigMap, but you can override this value  by adding or modifying the `autoscaling.knative.dev/target` annotation value in the revision template.
+
+Here is an example of how this annotation would be used in the revision template to set the limit to `50`.
```

Comment:
```suggestion
Here is an example of how this annotation would be used in the revision template to set the target to `50`.
```

Reply:
Added now and I've fixed this typo. Thanks!